packages/indy-vdr/tests/indy-vdr-pool.e2e.test.ts

```diff
@@ -10,9 +11,11 @@ import { IndyVdrPoolService } from '../src/pool/IndyVdrPoolService'
 import { indyDidFromPublicKeyBase58 } from '../src/utils/did'
 
 import { indyVdrModuleConfig } from './helpers'
+import '@hyperledger/aries-askar-nodejs'
 
 const indyVdrPoolService = new IndyVdrPoolService(testLogger, indyVdrModuleConfig)
-const wallet = new IndyWallet(agentDependencies, testLogger, new SigningProviderRegistry([]))
+const wallet = new AskarWallet(testLogger, new agentDependencies.FileSystem(), new SigningProviderRegistry([]))
```

Comment:
Do we want to start using askar in tests with the performance issues? CI is taking extremely long

Reply:
In this case, it does not affect so much because the bottleneck in these tests is the `@hyperledger/indy-vdr-nodejs` iself. But I think it might be better to just wait until we solve the ref-napi issue before replacing indy-sdk for tests.